x-pack/solutions/security/plugins/security_solution/public/common/components/visualization_actions/visualization_embeddable.tsx

```diff
@@ -145,12 +146,24 @@ const VisualizationEmbeddableComponent: React.FC<VisualizationEmbeddableProps> =
         donutTextWrapperClassName={donutTextWrapperClassName}
         donutTextWrapperStyles={donutTextWrapperStyles}
       >
-        <LensEmbeddable {...lensProps} id={id} onLoad={onEmbeddableLoad} />
+        <LensEmbeddable
+          signalIndexName={signalIndexName}
```

Comment:
I doubt any of the `lensProps` would override this value, but I would move it below the spread operator so it's not overwritten by accident. Same for the one below 😄